### PR TITLE
Update Contribution URL

### PR DESCRIPTION
**Pull request by mul14**
## Proposed changes

This PR just update Contribution URL, because the old URL no longer exists.

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/adonisjs/adonis-framework/blob/develop/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,4 +3,4 @@
 In favor of active development we accept contributions from everyone. You can contribute by submitting a bug, creating pull requests or even by improving documentation.
 
 Below is the guide to be followed strictly before submitting your pull requests.
-http://adonisjs.com/docs/3.1/contributing
+https://adonisjs.com/docs/4.1/contribution-guide
```
Comment (RomainLanz) [inline CONTRIBUTING.md]: It's better to remove the version from the link.

```suggestion
https://adonisjs.com/docs/contribution-guide
```

Comment (mul14) [reply CONTRIBUTING.md]: Sure, please wait.

Comment (mul14) [reply CONTRIBUTING.md]: Done, thank you for your feedback.

**Comment by RomainLanz** [comment]
Thanks 💜

**Pull request merged**